custom/enikshay/integrations/bets/utils.py

```diff
@@ -9,4 +26,8 @@ def get_bets_location_json(location):
         ancestor.location_type.name: ancestor.location_id
         for ancestor in location.get_ancestors()
     }
+    response['metadata'] = {
+        field: location.metadata.get(field) for field in
+        ['is_test', 'tests_available', 'private_sector_org_id', 'nikshay_code', 'enikshay_enabled']
+    }
```

Comment:
@proteusvacuum FYI, I'm freezing the locations API to what's described in the spec doc